### PR TITLE
Added MSW Logo hello world example

### LINKED ISSUE #169: Add MSW Logo

### PR DESCRIPTION
**Pull request by Kameramann23**
Closes #169

### CODE REVIEW

File: MSW Logo/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Hello World MSW Logo
+
+MSW Logo is an educational programming language.
+
+## Trings you need
```
Comment (sbshah97) [inline MSW Logo/README.md]: Trings?

**Review by sbshah97 — CHANGES_REQUESTED** [review]
Please create an Issue regarding this Pull Request and then in the Pull Request Description add the description `Closes #<Issue-Number>` to reference the Issue!

**Comment by sbshah97** [comment]
Any problems here?

**Comment by sbshah97** [comment]
@Kameramann23 any updates?

**Comment by Kameramann23** [comment]
Ill change it.

**Comment by Kameramann23** [comment]
Closes #169

**Review by sbshah97 — APPROVED** [review]

**Pull request merged**